components/match2/wikis/halo/match_group_input_custom.lua

```diff
@@ -284,7 +276,7 @@ function matchFunctions.readDate(matchArgs)
 		return dateProps
 	else
 		return {
-			date = _EPOCH_TIME_EXTENDED,
+			date = mw.getContentLanguage():formatDate('c', _EPOCH_TIME),
```

Comment:
This will return the same thing that `_EPOCH_TIME_EXTENDED` aleady is. Undo the change and you can remove a const.

Reply:
Just to confirm, undo which change?